test/quantization/fx/test_quantize_fx.py

```diff
@@ -2307,9 +2307,9 @@ def forward(self, x):
             "Expected the scripted model to preserve the state_dict for non-packed weight attributes")
         for attr_name in [
                 "mods1_0_input_scale_0", "mods1_0_input_zero_point_0",
-                "mods1_0_scale_0", "mods1_0_zero_point_0",
-                "mods1_1_scale_0", "mods1_1_zero_point_0",
-                "mods2_scale_0", "mods2_zero_point_0"]:
+                "mods1_0_output_scale_0", "mods1_0_output_zero_point_0",
+                "mods1_1_output_scale_0", "mods1_1_output_zero_point_0",
+                "mods2_output_scale_0", "mods2_output_zero_point_0"]:
```

Comment:
Note: This PR currently breaks backward compatibility since some scale and zp attributes currently have an extra `_output` prefix from [`quantize_node`](https://github.com/pytorch/pytorch/blob/d943a3e5a85b3ce4dbbadc2f3ac1cf4aa29d597b/torch/ao/quantization/fx/utils.py#L176), since the reference pattern code [calls that](https://github.com/pytorch/pytorch/blob/d943a3e5a85b3ce4dbbadc2f3ac1cf4aa29d597b/torch/ao/quantization/fx/quantization_patterns.py#L1065) and we now take this code path.

Reply:
(These changes are reverted. This PR no longer breaks BC)